firmware/controllers/algo/launch_control.cpp

```diff
@@ -134,21 +127,31 @@ bool LaunchControlBase::isLaunchConditionMet(int rpm) const {
 	return speedCondition && activateSwitchCondition && rpmCondition && tpsCondition;
 }
 
-void LaunchControlBase::update() {
+//void LaunchControlBase::update() {
```

Comment:
why "//void LaunchControlBase::update() {" dead code?

Reply:
I think what he meant to do is add a new function, `void updateLaunchConditions()` that simply does this:

```
void updateLaunchConditions() {
    launchInstance.update();
}
```